packages/amplify-sdk/src/index.js

```diff
@@ -177,18 +195,36 @@ export class AmplifySDK {
 					throw new TypeError('Expected org id');
 				}
 
-				await this.request('/api/v1/auth/switchLoggedInOrg', account, {
-					errorMsg: 'Failed to switch org',
-					json: { org_id: orgId }
-				});
+				if (account.auth.expired) {
+					account = await this.client.login();
+				}
 
-				// refresh the account
-				account = await this.auth.loadSession(account);
+				try {
+					await open(`https://platform.axway.com/#/switchorg/${orgId}`);
```

Comment:
So we're now going to be popping a browser just to switch orgs? I guess that's a workaround?

Reply:
Sadly, yes. I don't love it, especially in a library, but it's literally the only way.